### PR TITLE
[plugin.video.crossroads] 1.0.0

### PR DESCRIPTION
**Pull request by JoeKer1**
### Description
This is an addon to stream live and past services from www.crossroads.net/live

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.crossroads/addon.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.crossroads"
+version="1.0.0"
+name="Crossroads Church"
+provider-name="CRDS">
+<requires>
+  <import addon="xbmc.python" version="2.24.0"/>
```
Comment (enen92) [inline plugin.video.crossroads/addon.xml]: python api version should be 2.25.0 to match krypton (the repo to which you are submitting)

File: plugin.video.crossroads/addon.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <screenshot>resources\screenshot-01.jpg</screenshot>
+    <screenshot>resources\screenshot-02.jpg</screenshot>
+    <screenshot>resources\screenshot-03.jpg</screenshot>  
+  </assets>
+  <news>v1.0.0  (2017-1-27)
+- First version of Crossroads Stream Addon!
+- Added ability to stream live service if there is a stream active
+- Added ability to watch previous services</news>
+</extension>
+</addon>
```
Comment (enen92) [inline plugin.video.crossroads/addon.xml]: please include the <language> tag. See: http://kodi.wiki/view/addon.xml

**Comment by enen92** [comment]
Looks okay to me, please adjust my comments to merge

**Comment by JoeKer1** [comment]
@enen92 Thanks for the quick response. I think I got everything updated. let me know if I did it wrong or misunderstood your comment

**Pull request merged**